### PR TITLE
Describe how to configure the Minimum Qty Allowed in Shopping Cart

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes how to configure the Minimum Qty Allowed in Shopping Cart

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/cart.html

whatsnew
Added [Minimum Order Quantity](https://docs.magento.com/user-guide/sales/minimum-order-qty.html) topic to describe cart configuration for a minimum number of items per order.

### CODE REVIEW

File: src/sales/cart.md [hunk 1]
```diff
@@ -8,3 +8,26 @@ The cart is positioned at the end of the path to purchase, at the intersection o
...
+
+1. Expand **Product Stock Options** section.
+
+1. Set the minimum quantity in the **Minimum Qty Allowed in Shopping Cart** field.
+
+    To edit this field clear the **Use system value** checkboxes.
+
+    - Select Customer Group for setting minimum quantity limit for specific groups. To add another condition click **Add Minimum Qty**.
+
+    - To set a minimum quantity limit for all customers select **ALL GROUPS**.
```
Comment (leslietilling) [inline src/sales/cart.md:28]:
```suggestion
    - To set the same minimum quantity limit for all customers, keep the `ALL GROUPS` selection and enter the **Minimum Qty**.
```

File: src/sales/cart.md [hunk 2]
```diff
@@ -8,3 +8,26 @@ The cart is positioned at the end of the path to purchase, at the intersection o
...
+
+1. Select **Inventory** under **Catalog** section.
+
+1. Expand **Product Stock Options** section.
+
+1. Set the minimum quantity in the **Minimum Qty Allowed in Shopping Cart** field.
+
+    To edit this field clear the **Use system value** checkboxes.
+
+    - Select Customer Group for setting minimum quantity limit for specific groups. To add another condition click **Add Minimum Qty**.
```
Comment (leslietilling) [inline src/sales/cart.md:26]:
```suggestion
    - Change the **Customer Group** setting to a specific group and enter the **Minimum Qty** for that group. To add another group and quantity limit, click <span class="btn">Add Minimum Qty<span class="btn">.
```

File: src/sales/cart.md [hunk 3]
```diff
@@ -8,3 +8,26 @@ The cart is positioned at the end of the path to purchase, at the intersection o
...
+
+    - To set a minimum quantity limit for all customers select **ALL GROUPS**.
+
+1. Click **Save Config**.
```
Comment (leslietilling) [inline src/sales/cart.md:30]:
```suggestion
1. Click <span class="btn">Save Config</span>.
```

File: src/sales/cart.md [hunk 4]
```diff
@@ -8,3 +8,26 @@ The cart is positioned at the end of the path to purchase, at the intersection o
...
+
+    ![]({% link images/images/minimum-qty-allowed-in-shopping-cart.png %}){: .zoom}
+    _Minimum Qty Allowed to Order_
```
Comment (leslietilling) [inline src/sales/cart.md:33]:
```suggestion
    _Minimum quantity requirement in shopping cart_
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
I put in edits for the procedure -- this content is helpful but does not belong on this parent level page. I think we should create a new topic under https://docs.magento.com/user-guide/sales/cart-configuration.html (add the new topic to the table on that page), below the "Minimum Order Amount" topic. We should structure similar to that one with the example image of the storefront, and use a configuration example image within the procedure.

File: src/sales/cart.md (hunk 4 above)
Comment (ZhovnirMaria) [reply src/sales/cart.md]: Done, a new topic is created, please check🙏

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**